src/NUnitFramework/framework/Constraints/CollectionEquivalentConstraintResult.cs

```diff
@@ -51,13 +51,12 @@ public class CollectionEquivalentConstraintResult : ConstraintResult
             _tallyResult = tallyResult;
         }
 
-        /// <summary>Write the custom failure message for this object's <see cref="CollectionEquivalentConstraint"/>.</summary>
+         /// <summary>Write the custom extra chainable message to the failure message for a failing constraint. <see cref="CollectionEquivalentConstraint"/>.</summary>
         /// <param name="writer">The <see cref="MessageWriter"/> to write the failure message to.</param>
-        public override void WriteMessageTo(MessageWriter writer)
-        {
-            //Write the expected/actual message first.
-            base.WriteMessageTo(writer);
-            
+        public override void WriteExtraChainableMessage(MessageWriter writer)
+        {            
+            base.WriteExtraChainableMessage(writer);
```

Comment:
Since `base` is known to be `Constraint`, this line is guaranteed to be a no-op and should be removed.